site/src/components/Mainabout.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import '../sections/mainabout.css';
```

Comment:
@legendarykamal Please use Gatsby styled components instead of plane CSS.

Reply:
Yes @Chadha93 i had tried using it only at first place but faced a bit errors 😅 so shifted to plain ones. 
But I will consider converting this 👍 to styled ones

Reply:
@Chadha93 done changing to gatsby styled components